src/Plugin.cc

```diff
@@ -23,7 +25,7 @@ IGN_UTILS_WARN_IGNORE__DEPRECATED_DECLARATION
 #include "ignition/common/PluginInfo.hh"
 #include "PluginUtils.hh"
 
-namespace ignition
+namespace gz
```

Comment:
I think this broke the build, the namespace should remain `ignition` for plugin stuff. Maybe that makes it harder to do full sweeps of the code 😕

Reply:
https://github.com/gazebosim/gz-common/pull/356/commits/0a80e8ef969400f71429c7545ed1d3a722ce61c8

I'm surprised the namespace redirection didn't help fix this. Let's migrate fully to `gz` (since we have the namespace redirections in `config.hh` anyway)

Reply:
ok, fair enough, that should have pretty much the same result to users. The only downside is that they will get warnings saying that they should use `gz` instead of `ignition`, but `gz/Plugin.hh` doesn't exist.